addons/web/static/src/core/tooltip/tooltip_service.js

```diff
@@ -233,6 +233,12 @@ export const tooltipService = {
                     }
                 };
             },
+            show(el, params) {
+                openTooltip(el, params)
+                browser.setTimeout(() => {
+                    cleanup();
```

Comment:
needs a `target === el` check (with a long duration, the user could have opened another tooltip in the meantime, you don't want to close that other tooltip when this timeout comes through). Additionally, this tooltip may get removed prematurely because of the cleanup setInterval higher in this file. It sounds like in this case, you may just want to use the popover service directly in the user code instead of adding this capability to the service.

Reply:
Hm.... okay. I tought it was a good idea but I don't have any concrete arguments/needs. If it's better to not add that functionality, let's only use the popover service